/**:
  ros__parameters:
    # Node names to exclude from monitoring (Note: be careful with the "[node_name-#]" format)
    # Example: Do not issue a warning if rviz2 crashes.
    ignore_node_names:
      - rviz2

    # Exit codes to exclude from monitoring (e.g., Ctrl+C)
    # Example: 0, 130 are considered normal exits and not treated as errors.
    ignore_exit_codes:
      - 0
      - 130

    # Check interval (seconds)
    check_interval: 1.0

    # Enable/disable debug output
    enable_debug: false
